src/HeaderCell.tsx

```diff
@@ -224,8 +224,8 @@ export default function HeaderCell<R, SR>({
 
   function onDrop(event: React.DragEvent<HTMLDivElement>) {
     setIsOver(false);
-    if (event.dataTransfer.types.includes(dragDropKey)) {
-      const sourceKey = event.dataTransfer.getData(dragDropKey);
+    if (event.dataTransfer.types.includes(dragDropKey.toLowerCase())) {
```

Comment:
Could you add a comment explaining why `toLowerCase()` is necessary?
The comment can link to the spec.

Reply:
added